### PR TITLE
[CORDA-583] Test identities names cleanup

### PR DESCRIPTION
**Pull request by m4ksio**
Moved BigCorporation to NY for better visualisation

### CODE REVIEW

File: samples/bank-of-corda-demo/build.gradle [hunk 1]
```diff
@@ -74,7 +74,7 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
         ]
     }
     node {
-        name "O=BigCorporation,L=London,C=GB"
+        name "O=BigCorporation,L=New York,C=US"
```
Comment (al-r3) [inline samples/bank-of-corda-demo/build.gradle]: All of BigCorp changes seem not needed, it seems you can get away with just the change at line 62 here

Comment (m4ksio) [reply samples/bank-of-corda-demo/build.gradle]: Yes, but "moving" it do NY makes visualization in Explorer better

Comment (al-r3) [reply samples/bank-of-corda-demo/build.gradle]: ok, check that this doesn't affect any other demo though.

**Comment by al-r3** [comment]
This actually fixes a demo so it should be in before V1 is cut

**Pull request merged**